src/test/java/connectify/storage/JsonAdaptedCompanyTest.java

```diff
@@ -0,0 +1,20 @@
+package connectify.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import connectify.model.company.Company;
+import connectify.testutil.CompanyBuilder;
+
+public class JsonAdaptedCompanyTest {
+
+    private static final Company COMPANY_1 = new CompanyBuilder().withName("Company1").build();
+
+    @Test
+    public void toModelType_validCompanyDetails_returnsCompany() throws Exception {
+        JsonAdaptedCompany company = new JsonAdaptedCompany(COMPANY_1);
+        assertEquals(COMPANY_1, company.toModelType());
+    }
```

Comment:
Perhaps it would be better to add more tests for better code coverage?

Reply:
Thanks @victorpengmx, I held off on this first because I think we should discuss how we want to validate the company object, as some companies may have symbols inside their names, like 7-Eleven etc. 

I will update in the next interations.